open-media-match/.devcontainer/omm_config.py

```diff
@@ -1,6 +1,11 @@
 # Database configuration
+PRODUCTION = False
 DBUSER = 'media_match'
 DBPASS = 'hunter2'
 DBHOST = 'db'
 DBNAME = 'media_match'
 DATABASE_URI = f"postgresql+psycopg2://{DBUSER}:{DBPASS}@{DBHOST}/{DBNAME}"
+
+ROLE_HASHER = True
+ROLE_MATCHER = True
+ROLE_CURATOR = True
```

Comment:
(ignorable): The reason to split these into individual settings vs an enum is for the dev container to respond to all of these roles?

My initial reaction is if it was simpler to just have all roles have all APIs, and just route traffic based on roles, but this is simple and straightforward enough that I don't think that's needed.